.travis.yml

```diff
@@ -12,11 +12,12 @@ branches:
 
 services:
  - mongodb
+ - docker
 
 install:
  - npm install
 
 before_install:
+ - docker pull fiware/orion:0.26
```

Comment:
Orion 0.26 is pretty all... I'll sugest to use fiware:orion:latest. This may casuse some ocasionally breaking from time to time (as Orion may evolve) but I think is a good idea to be sure we keep backward compatibility as Orion evolves.

Reply:
Fixed in 0de8f251d1c82832cb5ee4b3104fd86100517c3d